### PR TITLE
Layout

### PR DESCRIPTION
**Pull request by KellerFuchs**

### CODE REVIEW

File: bylaws.md [hunk 1]
```diff
@@ -29,8 +32,8 @@ Section 1. Annual Meeting
 An annual meeting shall be held once each calendar year for the purpose
 of electing directors and for the transaction of such other business as
 may properly come before the meeting. The annual meeting shall be held
-at the time and place designated by the Board of Directors from time to
-time.
+at the time and place designated by the Board of Directors.
```
Comment (daurnimator) [inline bylaws.md]: 'from time to time' has been removed. please keep layout and content changes separate.

**Comment by daurnimator** [comment]
why?

I don't see much problem with the current layout.

File: bylaws.md (hunk 1 above)
Comment (KellerFuchs) [inline bylaws.md]: Yes, it is a separate commit.

**Comment by RyanSquared** [comment]
It's cleaner to read. 👍 from me.

**Comment by KellerFuchs** [comment]
@daurnimator Basically, humans are bad at reading walls of texts.
Breaking the text in paragraphs reduces the mental overhead involved.

**Comment by KellerFuchs** [comment]
OK, removed the commit @daurnimator didn't like.
@ChickenNuggers approved the layount change (and whitespace isn't legally significant), so I'm merging.

**Pull request merged**